proj/src/main/webapp/static/chat.html

```diff
@@ -99,7 +103,8 @@
                             <div id="button-house">
                                 <button type="button" id='friend-request' hidden></button>
                                 <button type="button" id='deny' hidden>deny friend request</button>
-                               <button type="button" id='block' hidden>block</button>
+                                <button type="button" id='block' hidden>block</button>
+
```

Comment:
Is this extra line here for any reason? Can it be removed to minimize the diff and reduce the chances that my incoming PR will have merge conflicts.